### DISCUSSION TITLE
'Select All' (CMD+A) or 'Copy' (CMD+C) disabled in RUPS versions 7.2.3 and above

### DISCUSSION THREAD

**Discussion by muscal** [itext/i7j-rups#42] [General]
As the title suggests, the Select and Copy functions (for manually selected fields in RUPS) no longer function on Mac OS 13.2. 

Had to go back to RUPS 7.2.2 to leverage that functionality again. 

![RUPS not working](https://user-images.githubusercontent.com/14338652/219490709-98d2d2bb-99ca-4c31-9729-94d0a82c932c.jpg)

**Comment by michaeldemey** [comment]
Hi!

Thanks for letting us know. We were able to reproduce this unwanted behavior. We'll see what the cause is for this.
